app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx

```diff
@@ -61,51 +63,64 @@ export class ManualJudgmentApproval extends React.Component<IProps, IState> {
 
   private handleStopClick = () => this.provideJudgment('stop');
 
+  private getInstructions = (): any => {
+    return {
+      __html: DOMPurify.sanitize(this.props.stage.context.instructions)
+    };
+  };
```

Comment:
Note: this is what the internet said is the way to safely render ad-hoc user-generated HTML.